pkgs/constants/constants.go

```diff
@@ -0,0 +1,57 @@
+//  constants.go
+//  https://github.com/apriorit/Soroka-EDMS/svc/users/pkgs/constants
+//
+//  Created by Ivan Kashuba on 2019.09.03
+//  Defines all known constant across all the packages
+package constants
+
+const (
+	RoleEndpoint          string = "/users/change_role"
+	ListEndpoint          string = "/users"
+	GetProfileEndpoint    string = "/user"
+	DisableUserEndpoint   string = "/users/disable/{id}"
+	EnableUserEndpoint    string = "/users/enable/{id}"
+	RegisterEndpoint      string = "/users/register"
+	IncorrectParameter    string = "Incorrect parameter %s"
+	MissingBody           string = "Missing content"
+	MissingPath           string = "Missing path"
+	MalformedBody         string = "Malformed body"
+	MalformedPath         string = "Malformed path"
+	EncodingErorr         string = "Enconding response error"
+	MissingPayload        string = "Missing payload"
+	InvalidAuthorization  string = "No priveledge"
+	InvalidSortParameters string = "Sort criteria is undefined"
+	ProfileNotFound       string = "No profile for such user id%d"
+	MalformedRequest      string = "Malformed request"
+	DisabledPartialy      string = "Disabled partialy"
+	EnabledPartialy       string = "Enabled partialy"
+	ChangedPartialy       string = "Changed partialy"
+	ChangedSuccessfully   string = "Successfully changed"
+	NoRecordChanged       string = "Nor record has been changed"
+	UserDoesNotExist      string = "User does not exist"
+	InvalidRole           string = "InvalidRole"
+	Invalid               string = "Invalid"
+	Required              string = "Required"
+	UserName_admin        string = "admin"
```

Comment:
Please move constants that are used only for stubbing directly to the stub implementation.